agorauikit_android/src/main/java/io/agora/agorauikit_android/AgoraSettings.kt

```diff
@@ -10,6 +10,9 @@ import io.agora.rtc.video.VideoEncoderConfiguration
 class AgoraSettings {
     public var userRtmMap = mutableMapOf<String, String>()
     public var uidToUserIdMap = mutableMapOf<Int, String>()
+
+    /// Whether RTM should be initialised and used
+    public var enableRtm: Boolean = false
```

Comment:
Let's keep this the same as the other platforms and integration doc `rtmEnabled`